gateway/src/apicast/policy/content_caching/Readme.md

```diff
@@ -0,0 +1,50 @@
+# APICast Policy caching
+
+This policy allows to enable and disable caching based on customized conditions. These
+conditions can only be applied on the client request, where upstream responses
+cannot be used in this policy.
+
```

Comment:
Should we clarify somewhere that this policy respects the cache headers set by upstream? By that, I mean that even if this policy is enabled and there's a rule that matches, a response will not be cached if the "Cache-control" header is set to "no-cache" by the upstream.